NugetLightRequest.cs

```diff
@@ -1594,28 +1731,35 @@ private bool TryParseFastPath(string fastPath, out string source, out string id,
             return match.Success;
         }
 
-        private bool LocationCloseEnoughMatch(string givenLocation, string knownLocation) {
-            if (givenLocation.Equals(knownLocation, StringComparison.OrdinalIgnoreCase)) {
+        private bool LocationCloseEnoughMatch(string givenLocation, string knownLocation)
+        {
+            if (givenLocation.Equals(knownLocation, StringComparison.OrdinalIgnoreCase))
+            {
                 return true;
             }
 
             // make trailing slashes consistent
-            if (givenLocation.TrimEnd('/').Equals(knownLocation.TrimEnd('/'), StringComparison.OrdinalIgnoreCase)) {
+            if (givenLocation.TrimEnd('/').Equals(knownLocation.TrimEnd('/'), StringComparison.OrdinalIgnoreCase))
+            {
                 return true;
             }
 
             // and trailing backslashes
-            if (givenLocation.TrimEnd('\\').Equals(knownLocation.TrimEnd('\\'), StringComparison.OrdinalIgnoreCase)) {
+            if (givenLocation.TrimEnd('\\').Equals(knownLocation.TrimEnd('\\'), StringComparison.OrdinalIgnoreCase))
+            {
                 return true;
             }
             return false;
         }
-
-        private IEnumerable<IPackage> FilterOnVersion(IEnumerable<IPackage> pkgs, string requiredVersion, string minimumVersion, string maximumVersion, bool minInclusive = true, bool maxInclusive = true) {
+        /*
```

Comment:
>/* [](start = 8, length = 2)

If below commented code is not required anymore, consider removing.

Reply:
Cleaning up

---
In reply to: [154509959](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154509959) [](ancestors = 154509959)